src/utils/errors.py

```diff
@@ -71,27 +50,9 @@ def not_found(self, msg: str = "Content not found") -> ErrorProps:
             ),
         )
 
-    def content_exists(self, msg: str = "This content exists already") -> ErrorProps:
-        log.error(f"{status.HTTP_409_CONFLICT}: {str(msg)}")
-        return JSONResponse(
-            status_code=status.HTTP_409_CONFLICT,
-            content=jsonable_encoder(
-                {"error": str(msg), "status_code": status.HTTP_409_CONFLICT}
-            ),
-        )
-
-    def content_gone(self, msg: str = "Content gone") -> ErrorProps:
-        log.error(f"{status.HTTP_410_GONE}: {str(msg)}")
-        return JSONResponse(
-            status_code=status.HTTP_410_GONE,
-            content=jsonable_encoder(
-                {"error": str(msg), "status_code": status.HTTP_410_GONE}
-            ),
-        )
-
     def unknown_error(
         self, msg: str = "An unknown error was encountered"
-    ) -> ErrorProps:
+    ) -> ErrorProps:  # pragma: no cover
```

Comment:
@sinnec how so?

Reply:
I figured I shouldn't remove it since it's an unknown error, but on the other hand it should be skipped in the coverage report since it can't be reproduced on a fully working application. 🤷🏻‍♂️

Reply:
Where is this used?

Reply:
How should I know? It's unkown! 😂
Joking aside, do you think this should be removed too?

Reply:
I search for a reference but didn't find any. If it is not used we can remove it

Reply:
Removed